kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -94,33 +94,43 @@ class ResidualBasedBDFCustomScheme
     ///@name Life Cycle
     ///@{
 
+    /**
+     * @brief Constructor. The BDF method
+     * @param ThisParameters The parameters containing the list of variables to consider
+     * @todo The ideal would be to use directly the dof or the variable itself to identify the type of variable and is derivatives
+     */
+    explicit ResidualBasedBDFCustomScheme(Parameters ThisParameters)
+        : ResidualBasedBDFCustomScheme(ThisParameters.Has("integration_order") ? static_cast<std::size_t>(ThisParameters["integration_order"].GetInt()) : 2, ThisParameters)
```

Comment:
Explanation is below

Reply:
validate and then call the constructor from the constructor body (it is admissible in c++11)

Reply:
you did not address my concern here...

Reply:
It is not possible, because is a derived class, which already works with parameters, it is not possible without doing a complete mess and a lot of meaningless code duplication. I think this is the cleanest solution possible